docs_source_files/content/webdriver/mouse.de.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Maus"
+weight: 11
+---
+
```

Comment:
Hi @boris779,
This section is removed from WebDriver and placed in support packages I think. This should be removed i think

Reply:
of course, I will remove that